client/my-sites/plugins/index.js

```diff
@@ -102,6 +103,17 @@ export default function () {
 			clientRender
 		);
 
+		if ( config.isEnabled( 'marketplace-yoast' ) ) {
+			page(
+				'/plugins/domain/:site?',
+				scrollTopIfNoHash,
```

Comment:
Do we really need this? Are we expecting to use hash for anything on this new page?

Reply:
Fixed by 0aa0786